notebooks/parler-tts-text-to-speech/parler-tts-text-to-speech.ipynb

```diff
@@ -0,0 +1,580 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    %pip uninstall -q -y openvino-dev openvino openvino-nightly
</pre><p>please use one line instead of uninstall + install:</p>

<pre class="ql-syntax" spellcheck="false">%pip install --pre -Uq openvino --extra-index-url https://storage.openvinotoolkit.org/simple/wheels/nightly
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2114/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>please also add lower bound for torch and gradio</p>

Reply:
<p>fixed</p>